File: 485-max-consecutive-ones/485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let arr = [];
    let count = 0;
    for (let i = 0; i < nums.length; i++){
        if (nums[i] === 1){
            count++
        } else {
            arr.push(count);
            count = 0;
        }
        arr.push(count);
    }
    arr.sort((a,b) => b - a);
    return arr[0];
};

File: 485-max-consecutive-ones/485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxi = 0
        count = 0
        for i in nums:
            if i == 1:
                count += 1
                if maxi < count:
                    maxi = count
            else:
                count = 0
        return maxi